scripts/create-kubestellar-demo-env.sh

```diff
@@ -13,10 +13,39 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Deploys Kubestellar environment for demo purposes.
+# Deploys Kubestellar environment for demo purposes on kind or k3d.
 
 set -e
 
+# Default Kubernetes platform parameter
+k8s_platform="kind"
+
+# Parse command-line arguments
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --platform) k8s_platform="$2"; shift ;;
+        -X) set -x ;;
+        -h|--help)
+            echo "Usage: $0 [--platform <kind|k3d>] [-X] [-h|--help]" >&2
+            exit 0
+            ;;
+        *)
+            echo "Unknown parameter passed: $1"
```

Comment:
FYI, this seems like an error message to me, something that I would write to stderr.